server/mongodb/workersDB.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require('mongoose');
+
+// create schema
+const workerSchema = new mongoose.Schema({
+    id: String,
+    name: String,
+    email: String,
+    phone: String,
+    address: String,
+    hRate: Number,
+    trades: String,
+    pCode: String,
```

Comment:
i prefer `postalCode` instead of `pCode` as i think the former is more explicit

same thing for the `hRate`, we can have `hourlyRate`